docs/pages/debugging/tools.mdx

```diff
@@ -205,11 +205,16 @@ Pressing the <kbd>shift</kbd> + <kbd>m</kbd> shortcut will also show installed [
 
 ## React Native Debugger
 
-> **warning** The React Native Debugger does not support apps using Hermes, see the [Chrome DevTools](#debugging-with-chrome-devtools) section.
+> **warning** The React Native Debugger requires Remote JS debugging, which has been deprecated since [React Native 0.73](https://reactnative.dev/docs/other-debugging-methods#remote-javascript-debugging-deprecated).
+> The React Native Debugger is a standalone app that wraps the React DevTools, Redux DevTools, and the Chrome DevTools. Unfortunately, [React Native Debugger requires](https://github.com/jhen0409/react-native-debugger/discussions/774) the deprecated Remote JS debugging workflow and is incompatible with Hermes.
 
-The React Native Debugger includes many tools listed later on this page, all bundled into one, including [React DevTools](#debugging-with-react-devtools) and network request inspection. For this reason, if you use one tool from this page, it should probably be this one.
+If you are using Expo **SDK 50** or **later**, you can use the [Expo Devtool plugins](/debugging/devtools-plugins) equivalents to the React Native Debugger.
 
-We'll give a quick look at it here, but check out their [documentation](https://github.com/jhen0409/react-native-debugger#documentation) for a more in-depth look.
+- [React DevTools](#debugging-with-react-devtools)
+- [Redux DevTools](/debugging/devtools-plugins/#redux)
+- [Chrome DevTools](#debugging-with-chrome-devtools)
+
+If you are using Expo **SDK 49** or **earlier**, you can use the React Native Debugger. We'll give a quick look at it here, but check out their [documentation](https://github.com/jhen0409/react-native-debugger#documentation) for a more in-depth look.
```

Comment:
```suggestion
If you are using Expo SDK 49 or below, you can use the React Native Debugger. This section provides quick get started instructions. For in-depth information, check its [documentation](https://github.com/jhen0409/react-native-debugger#documentation).
```

Improve the readability. We also don't need to use first person voice since the second sentence can be more direct about what this section provides and where to look for more information.